spyder/plugins/application/widgets/status.py

```diff
@@ -106,11 +118,15 @@ def get_icon(self):
     def start_installation(self, latest_release):
         self.installer.start_installation(latest_release)
 
+    def set_download_progress(self, current_value, total):
+        percentage_progress = 0
+        if total > 0:
+            percentage_progress = int((current_value/total) * 100)
```

Comment:
```suggestion
            percentage_progress = round((current_value/total) * 100)
```

Perhaps slightly more idiomatic (and perhaps a tad more representative, since it rounds rather than truncates)